packages/@aws-cdk/aws-lambda-go/test/lambda-handler-vendor/go.mod

```
@@ -4,7 +4,13 @@ go 1.16
 
 require (
 	github.com/aws/aws-lambda-go v1.19.1
+	github.com/go-check/check v0.0.0-20200902074654-038fdea0a05b // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
-	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
+)
+
+replace (
+	gopkg.in/check.v1 => github.com/go-check/check v0.0.0-20200902074654-038fdea0a05b
```

This should point to the `v1` branch of the repo. I think:

```suggestion
	gopkg.in/check.v1 => github.com/go-check/check v1
```